configure.ac

```diff
@@ -5069,6 +5071,59 @@ PKG_CHECK_MODULES([LIBSIGROK], [libsigrok < 0.4],
 )
 # }}}
 
+# --with-libssl {{{
+with_libssl_cflags=""
+with_libssl_ldflags=""
+AC_ARG_WITH(libssl, [AS_HELP_STRING([--with-libssl@<:@=PREFIX@:>@], [Path to libssl.])],
```

Comment:
From here on, can you please keep the style of the existing configure.ac? So quote all macro arguments, if; then on the same line etc.

Reply:
Fixed.